component/newstorage/batchedstore/batchedstore_test.go

```diff
@@ -7,76 +7,19 @@ SPDX-License-Identifier: Apache-2.0
 package batchedstore_test
 
 import (
-	"encoding/base64"
 	"errors"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 
 	"github.com/hyperledger/aries-framework-go/component/newstorage/batchedstore"
 	"github.com/hyperledger/aries-framework-go/component/newstorage/formattedstore"
+	"github.com/hyperledger/aries-framework-go/component/newstorage/formattedstore/exampleformatters"
 	"github.com/hyperledger/aries-framework-go/component/newstorage/mem"
 	"github.com/hyperledger/aries-framework-go/component/newstorage/mock"
-	"github.com/hyperledger/aries-framework-go/pkg/newstorage"
 	newstoragetest "github.com/hyperledger/aries-framework-go/test/newstorage"
 )
 
-type base64Formatter struct {
-}
-
-func (b *base64Formatter) FormatKey(key string) (string, error) {
-	return base64.StdEncoding.EncodeToString([]byte(key)), nil
-}
-
-func (b *base64Formatter) FormatValue(value []byte) ([]byte, error) {
-	return []byte(base64.StdEncoding.EncodeToString(value)), nil
-}
-
-func (b *base64Formatter) FormatTags(tags ...newstorage.Tag) ([]newstorage.Tag, error) {
-	formattedTags := make([]newstorage.Tag, len(tags))
-
-	for i, tag := range tags {
-		formattedTags[i] = newstorage.Tag{
-			Name:  base64.StdEncoding.EncodeToString([]byte(tag.Name)),
-			Value: base64.StdEncoding.EncodeToString([]byte(tag.Value)),
-		}
-	}
-
-	return formattedTags, nil
-}
-
-func (b *base64Formatter) DeformatKey(formattedKey string) (string, error) {
```

Comment:
Moved these to a separate package since they're shared across a few packages now